src/fragments/start/getting-started/react/auth.mdx

```diff
@@ -28,53 +28,34 @@ amplify console
 
 ## Create login UI
 
-Now that we have our authentication service deployed to AWS, it's time to add authentication to our React app. Creating the login flow can be quite difficult and time consuming to get right. Luckily Amplify Framework has an authentication UI component we can use that will provide the entire authentication flow for us, using our configuration specified in our __aws-exports.js__ file.
+Now that we have our authentication service deployed to AWS, it's time to add authentication to our React app. Creating the login flow can be quite difficult and time consuming to get right. Luckily Amplify Framework has an [authentication](https://ui.docs.amplify.aws/react/components/authenticator) UI component we can use that will provide the entire authentication flow for us, using our configuration specified in our **aws-exports.js** file.
 
-Open __src/App.js__ and make the following changes:
+Open **src/App.js** and make the following changes:
 
-1. Import the `Authenticator` component:
+1. Import the `withAuthenticator` component:
 
 ```javascript
-import { Authenticator } from '@aws-amplify/ui-react';
+import { withAuthenticator } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 ```
 
-2. Wrap your application with the `Authenticator` component. Once a user has created an account and signed in, the underlying component will be rendered with access to the `user`.
+2. Wrap your application with the `withAuthenticator` component.
 
 ```javascript
-return (
-  <Authenticator>
-    {({ signOut, user }) => (
-      <div style={styles.container}>
-        <h1>Hello {user.username}</h1>
-        <button style={styles.button} onClick={signOut}>Sign out</button>
-        <br />
-        <h2>Amplify Todos</h2>
-        <input
-          onChange={event => setInput('name', event.target.value)}
-          style={styles.input}
-          value={formState.name}
-          placeholder="Name"
-        />
-        <input
-          onChange={event => setInput('description', event.target.value)}
-          style={styles.input}
-          value={formState.description}
-          placeholder="Description"
-        />
-        <button style={styles.button} onClick={addTodo}>Create Todo</button>
-        {
-          todos.map((todo, index) => (
-            <div key={todo.id ? todo.id : index} style={styles.todo}>
-              <p style={styles.todoName}>{todo.name}</p>
-              <p style={styles.todoDescription}>{todo.description}</p>
-            </div>
-          ))
-        }
-      </div>
-    )}
-  </Authenticator>
-);
+/* src/App.js */
+function App({ signOut, user }) {
+  // Todo logic here
+
+  return (
+    <>
+      {/* Add Todo JSX here  */}
+      <h1>Hello {user.username}</h1>
+      <button onClick={signOut}>Sign out</button>
```

Comment:
Just an idea: would we want to demonstrate how to import and use our own UI components such as Heading and Button?

Reply:
OK I updated it!